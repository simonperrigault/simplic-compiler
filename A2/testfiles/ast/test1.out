Program
  List
    FuncDecl
      IdDecl ID="main"
      List
        FuncParam
          IdDecl ID="param1"
      List
      Opt
    FuncDecl
      IdDecl ID="main_2"
      List
      List
      Opt
        ReturnStmt
          Numeral NUMERAL="0"
