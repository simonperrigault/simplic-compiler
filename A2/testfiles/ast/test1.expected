Program
  List
    FuncDecl
      IdDecl ID="main"
      List
        FuncParam
          IdDecl ID="param1"
      List
    FuncDecl
      IdDecl ID="main_2"
      List
      List
        ReturnStmt
          Numeral NUMERAL="0"